### PR TITLE
ShipStation Connector (Independent Publisher)

### PR DESCRIPTION
**Pull request by SmolSoftBoi**
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thank you @SmolSoftBoi for creating the ShipStation connector! :) Welcome to the Independent Publisher connector community! We will be reviewing your connector in the next 2 weeks.

**Comment by SmolSoftBoi** [comment]
@natalie-pienkowska I can't see the issues with the API properties?

File: independent-publisher-connectors/ShipStation/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../../schemas/paconn-apiProperties.schema.json",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/ShipStation/apiProperties.json:2]: @SmolSoftBoi Can you please let me know the reason for adding this property? This is neither allowed nor valid, Can you please make sure that you are validating your connector artifacts with paconn tool.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/ShipStation/readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Prerequisites
+ShipStation uses the ISO 8601 combined format for DateTime stamps being submitted to and returned from the API. Please be sure to submit all DateTime values as follows:
+
+yyyy-mm-dd hh:mm:ss (24 hour notation). Example: 2016-11-29 23:59:59
+
+The time zone represented in all API responses is PST/PDT. Similarly, ShipStation asks that you make all time zone convertions and submit any DateTime requests in PST/PDT.
+
+## Supported Operations
+Required. Describe actions, triggers, and other endpoints.​
+
```
[the remark this answers is not in the document]
Comment (Amjed-Ayoub) [reply independent-publisher-connectors/ShipStation/readme.md:21]: Hi @SmolSoftBoi, can you please add the required fields in the readme file. please follow [this template](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/Independent%20Publisher).
Thank you very much.

File: independent-publisher-connectors/ShipStation/apiProperties.json (hunk 1 above)
Comment (Amjed-Ayoub) [reply independent-publisher-connectors/ShipStation/apiProperties.json]: Hi @SmolSoftBoi, I am following up to check on how updating your artifacts are going.
Please let me know if you have any questions.
Thank you very much

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hi @SmolSoftBoi thank you for submitting your connector with us. The request is approved, and the certification process has started for your connector. this process takes 5 to 6 weeks to complete.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @SmolSoftBoi I hope you are doing well.
When we tried to add your connector to ISV studio it turned out that this name "ShipStation" is used by an existing connector. Can you please change the name of your connector? maybe to "ShipStationIP".
After you change the name, please submit a new commit and we will approve it so we can move forward with the certification process.
Please let me know if you have any questions.
Apologies for the inconvenience.
Thank you very much.

**Comment by SmolSoftBoi** [comment]
> When we tried to add your connector to ISV studio it turned out that this name "ShipStation" is used by an existing connector. Can you please change the name of your connector? maybe to "ShipStationIP".

What's `ISV studio`? I can't see any existing connectors with the name `ShipStation`.

**Comment by Amjed-Ayoub** [comment]
Hi @SmolSoftBoi  I hope you are doing well. I am following up with you to see how things are going with updating your connector artifacts.
Please let us know if you have any questions.
We look forward to getting your connector certified.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @SmolSoftBoi,
We have ShipStation as an internal connector. apologies for the inconvenience this may have caused. We can't have two connectors with the same name. 
ISV Studio is where your connector will live during the certification process and after the process is complete and the connector is deployed to all our regions it will be available [here](https://us.flow.microsoft.com/en-us/connectors/). We will create the connector is ISV Studio for you. :-)
Please just make a small change to the title of your connector maybe just add IP at the end of it, so we can move forward with certifying your connector.
Please let me know if you have any questions.
Thank you very much.

**Comment by SmolSoftBoi** [comment]
> Please just make a small change to the title of your connector maybe just add IP at the end of it, so we can move forward with certifying your connector.

Which reference(s) actually need changing?

File: independent-publisher-connectors/ShipStation/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,398 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "ShipStation",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/ShipStation/apiDefinition.swagger.json:5]: Please change the title here.

File: independent-publisher-connectors/ShipStation/readme.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# ShipStation
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/ShipStation/readme.md:1]: Please change the title in the readme as well

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hi @SmolSoftBoi, thank you so much for the update. 
May I ask you please to change the folder name to ShipStationIP as well. Please see below
image
Apologies for the multiple requests.
Thank you very much for your understanding.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @SmolSoftBoi,
I hope you are doing well.
Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @SmolSoftBoi,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by sriyen-msft** [comment]
@SmolSoftBoi Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

File: independent-publisher-connectors/ShipStationIP/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,398 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "ShipStation IP",
+    "description": "The API is a great way to get data directly to and from ShipStation, like creating orders, updating products, and querying order, shipment and customer data.",
+    "contact": {
+      "name": "ShipStation",
+      "url": "https://shipstation.com"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/ShipStationIP/apiDefinition.swagger.json:9]: Hello @SmolSoftBoi,
I hope you are doing well.
Can you please add an email address? We have a weekly connector telemetry report for our independent publishers. To start getting this report, please add your email in the swagger file and submit a quick update.
Please let me know if you have any questions.
Thank you so much for you hard work with us.
Best regards,
Amjed Ayoub

Comment (SmolSoftBoi) [reply independent-publisher-connectors/ShipStationIP/apiDefinition.swagger.json:9]: Added in pull request #1813

**Pull request merged**